biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -1199,16 +1205,39 @@ public void report(PrintStream out) {
 					out.println("Id    Levl State Modified      Location");
 
 					for (int i = 0; i < bundles.length; i++) {
+						String lastModified = "<>";
 						String loc = bundles[i].getLocation();
-						loc = loc.replaceAll("\\w+:", "");
-						File f = new File(loc);
+						try {
```

Comment:
I think this warrants a method, maybe in `aQute.libg.uri.URIUtil`?

Reply:
:+1: Will do.